spec/report_spec.rb

```diff
@@ -19,9 +19,9 @@
           response = described_class.retrieve(is24_id, Date.today, Date.today)
 
           expect(response.code).to eq '200'
-          expect(response.messages).to eq({"dailyReports"=>{"@objectNumber"=>"S3W0UP6C", "reportDailyData"=>[{"date"=>"2019-06-11", "matchesResultList"=>0,
-            "displaysResultList"=>0, "exposeHits"=>0, "onShortList"=>0, "clicksHomepage"=>0, "emailContacts"=>0, "clicksSendUrl"=>0, "clickFocusPlacement"=>0,
-            "showMiniExposeFocusPlacement"=>0, "displayFocusPlacement"=>0}]}})
+          expect(response.messages).to eq({"dailyReports"=>{"objectNumber"=>"S3W0UP6C", "reportDailyData"=>{"date"=>"2019-06-11", "matchesResultList"=>"0",
+            "displaysResultList"=>"0", "exposeHits"=>"0", "onShortList"=>"0", "clicksHomepage"=>"0", "emailContacts"=>"0", "clicksSendUrl"=>"0", "clickFocusPlacement"=>"0",
+            "showMiniExposeFocusPlacement"=>"0", "displayFocusPlacement"=>"0"}}})
```

Comment:
Does this have implications in our consumers? I think we should still make sure to return numbers here, even if the parser gives us strings.

 cc @wjfrey

Reply:
I'm pretty sure this doesn't have implications, we are just saving `expose_hits` and this field is an integer in the database

Reply:
This should go in the changelog as a breaking change. I would personally still prefer if the result data just stayed the same.

Reply:
Yes, good point.

Reply:
> This should go in the changelog as a breaking change.

Sorry, I'm not sure I understand this line...It means we should document this for the other potential user gem users somewhere?